rust/parquet/src/file/writer.rs

```diff
@@ -114,6 +114,7 @@ pub trait RowGroupWriter {
 // ----------------------------------------------------------------------
 // Serialized impl for file & row group writers
 
+pub use crate::util::io::TryClone;
```

Comment:
I'd prefer if the import is still close to line 41. Here it's hidden, especially as a `pub use`

Reply:
moved!